tests/fixtures/token_factory.py

```diff
@@ -1,21 +1,31 @@
-import factory
-from factory import lazy_attribute
+import re
+
+from factory import Factory, lazy_attribute
 from faker import Faker
 
-from domain.tx.token import TokenMetadata, TokenNFT, TokenNFTType
+from domain.tx.token import Token, TokenMetadata, TokenNFT, TokenNFTType
 
 fake = Faker()
 
 
-class TokenNFTFactory(factory.Factory):
+def token_symbol(name):
+    first_name = name.split(' ')[0]
+    symbol = re.sub(r'[aeiou\s]', '', first_name).upper()[:5]
+    if len(symbol) < 3:
```

Comment:
Why do we need this? I feel we should never change the token name, but I don't have the full picture here. Anyway, add a comment with an example to clarify please.

Reply:
@msbrogli It's just a factory for test purpose. I just tried to make it a little more realistic.